homework3/EVPurchaseProgram2/src/main/java/ro/siit/java10/evp/CarDealership.java

```diff
@@ -0,0 +1,133 @@
+package ro.siit.java10.evp;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+
+public class CarDealership {
+
+
+    private String name;
+    private Repository repository;
+    private double budget;
+
+    public CarDealership(String name) {
+        this.name = name;
+        repository = new Repository();
+        budget = 0;
+    }
+
+    public ArrayList<DatabaseElement> getCarsList() {
+        return repository.getCarsList();
+    }
+
+    public ArrayList<DatabaseElement> filterCarsByFastCharging() {
+
+        return CarFilter.filterCarsByFastCharging(repository.getCarsList());
+    }
+
+    public ArrayList<DatabaseElement> filterCarsByStock() {
+
+        return CarFilter.filterCarsByStock(repository.getCarsList());
+    }
+
+
```

Comment:
no need for the second blank line.
Please read Java Code Convention Style - Oracle